source/manage-contacts.rst

```diff
@@ -0,0 +1,33 @@
+==============================
+Manage Contacts: Contacts View
+==============================
+
+To manage contacts in PodBrowser, switch to the :guilabel:`Contacts`
```

Comment:
Seems repetitive.  Maybe "To manage contacts in PodBrowser, switch to the :guilabel:`Contacts` view by clicking on the Contacts icon on the top menu bar."